manifests/0000_90_etcd-operator_03_prometheusrule.yaml

```diff
@@ -120,6 +121,7 @@ spec:
         severity: critical
     - alert: etcdExcessiveDatabaseGrowth
       annotations:
+        summary: etcd cluster received 50 percent increase in database writes.
```

Comment:
I think the alert triggers when the db size increased by 50%, not the number of writes

Reply:
👍🏽